### DISCUSSION TITLE
Analog dial arm can obscure digits.

### DISCUSSION THREAD

**Discussion by cizzop** [jomjol/AI-on-the-edge-device#3350] [Q&A] [answered]
Hello,

I am looking to monitor my water meter but I noticed that my meter has an analog dial arm that will overlap the digits at times. Will this work with ai-on-the-edge? I can't seem to find anything in the documentation in regards to this. 

See attached image here:

![meter](https://github.com/user-attachments/assets/72763b0d-d03a-4b89-9ca8-84130aa496ff)

**Comment by dwightc** [comment, marked as answer]
Yes, AI-on-the-edge will work with this style of meter, but it might take some fine-tuning to get satisfactory results. I have a meter that is similar to yours and I have been using it for the past eight months with pretty good results. It hasn't been perfect, but I have it to the point where it will often run for months at a time without requiring any attention from me.

I had to set the analog detection zone to be pretty large, but it's usually within a couple hundredths.

The settings I have found to be the most helpful are:

- Setting "Allow Negative Rates" to false, since this is a water meter and theoretically should never run backwards.
- Setting "Maximum Rate Type" to "AbsoluteChange" and "Maximum Rate Value" to a fairly low value.

I have set them thus because when a meter read is taken while the needle is sweeping over the digits it will often get them wrong, resulting in a reading that is way off. You want these incorrect values to be discarded.

Here's a shot of my meter:
![image](https://github.com/user-attachments/assets/74ae51bb-a1c0-4699-b0ce-d005e5234b0c)

... and here's a link to me asking the very same question eight months ago: https://github.com/jomjol/AI-on-the-edge-device/discussions/3003

Edit: Added info about analog detection and fixed broken URL.

**Reply by Caddyman68** [reply]
I'm glad I found your answer in this forum. I have already ordered everything I think I need for the project including external antenna supplies since meter is buried and 120 feet from router. All the examples I saw prior to this showed what I would call a digital meter - no needle. Mine is a model C700 (also Invision) like yours but says manufactured by Elster. Looks very close to yours. Thanks for posting

**Comment by cizzop** [comment]
@dwightc Thank you so much for your reply. I will build a mount and try this out since it worked for you. Great tips on those settings!

**Comment by cizzop** [comment]
@dwightc I got this working rather easily so far. However, I didn't bother adding the analog needle as I don't really need that much precision. Thanks again for the advice. 

Just curious, what do you set your maximum rate value to? I have mine set to 5 but I haven't done anything so far that uses a large amount of water over a short amount of time. edit: I should add that my update is every 3 minutes.

**Reply by dwightc** [reply]
I have my Maximum Rate set to 1.33, and the Auto Timer set to run the detection every minute. It's able to run that frequently because I've set it up to skip the Alignment step, since my camera is physically aligned in a rigid mount.

It was a tradeoff. I wanted to set the Maximum Rate low to keep the software from getting too far ahead of the actual meter in the event of a misread. When this happens it will show a large usage of water and then zero usage until the meter "catches up" to the misread value, since I have negative values disabled.

However, if I set it too low there was a chance that the water usage could legitimately outrun my Maximum Rate and then the software would become stuck. This was especially true if I ended up with misreads on back-to-back readings. I had initially set it to 1.0, but this turned out to be too low.

It turns out that 1.33 cubic feet per minute (about 10 gallons) is more than my household ever seems to use, and so I have had no trouble for months.

There are different settings that can work based more upon the rate-of-change of the meter instead of absolute values, but the ones I have chosen worked so well that I haven't played with them at all.
